_episodes/03-lists.md

```diff
@@ -38,20 +38,23 @@ odds are: [1, 3, 5, 7]
 ~~~
 {: .output}
 
-We select individual elements from lists by indexing them:
+We can select individual elements from lists by indexing them.
```

Comment:
While we're at it, let's improve this sentence.

> We can select individual elements from lists by indexing them.

There are two problems with it: we're not selecting elements (as you correctly wrote below, we're **accessing** them), and we're not indexing them (indexing == the process of assigning indices).

I suggest saying something like this:

> We can access elements of a list using indices -- numbered positions of elements in the list. These positions are numbered starting at 0, so the first element has an index of 0.

Reply:
Done